sbatches/farmshare/singularity-jupyter.sbatch

```diff
@@ -16,7 +16,7 @@
 
 PORT=$1
 NOTEBOOK_DIR=${2:-/farmshare/scratch/users/$USER}
-CONTAINER=${3:-/farmshare/home/classes/bioe/301p/ce/ces}
+CONTAINER=${3:-$HOME/image_latest.sif}
```

Comment:
The docker URI could just as easily be here. And can we use a standard, well maintained image?

Reply:
Docker does not work on farmshare - to the best of my knowledge. 

It supports singularity, and as the image must be downloaded to the farmshare.

Reply:
`docker://jupyter/scipy-notebook` is a singularity resource identifier.

Reply:
Changed as of 
0341779